src/omnicore/tx.h

```diff
@@ -217,7 +221,9 @@ class CMPTransaction
         blocktimelimit = 0;
         min_fee = 0;
         subaction = 0;
-        memset(&alertString, 0, sizeof(alertString));
+        alert_type = 0;
+        alert_expiry = 0;
+        memset(&alert_text, 0, sizeof(alert_text));
```

Comment:
My mistake of introducing this in the first place. Since it's a std::string, and not a char array, we can and should just use `alert_text.clear()`.